### PR TITLE
Creating types for plotly.js-dist-min package

### PR DESCRIPTION
**Pull request by andrefarzat**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "dtslint/dt.json" }`, and no additional rules.
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/plotly.js-dist-min/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+// Type definitions for plotly.js-dist-min 2.3
+// Project: https://github.com/plotly/plotly.js#readme
+// Definitions by: André Farzat <https://github.com/andrefarzat>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Forked from: plotly.js-basic-dist by Florian Dreier <https://github.com/DreierF>
+
+import * as Plotly from 'plotly.js';
+
+export default Plotly;
```
Comment (amcasey) [inline types/plotly.js-dist-min/index.d.ts:9]: Plotly appears not to have a default export, so this should probably be `export = Plotly` as in https://github.com/DefinitelyTyped/DefinitelyTyped/blob/8d85752375d30df9c82a94af9525764581e28ca3/types/plotly.js-basic-dist/index.d.ts#L8.

File: types/plotly.js-dist-min/plotly.js-dist-min-tests.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import Plotly from 'plotly.js-dist-min';
```
Comment (amcasey) [inline types/plotly.js-dist-min/plotly.js-dist-min-tests.ts:1]: This should probably be `import * as Plotly`.

**Review by amcasey — CHANGES_REQUESTED** [review]

File: types/plotly.js-dist-min/index.d.ts (hunk 1 above)
Comment (andrefarzat) [reply types/plotly.js-dist-min/index.d.ts]: Fixed 😺

File: types/plotly.js-dist-min/plotly.js-dist-min-tests.ts (hunk 2 above)
Comment (andrefarzat) [reply types/plotly.js-dist-min/plotly.js-dist-min-tests.ts]: Fixed 😺

**Review by amcasey — APPROVED** [review]
Thanks!

**Comment by andrefarzat** [comment]
@amcasey thanks a lot for your review! 😸

**Comment by andrefarzat** [comment]
Ready to merge

**Pull request merged**